<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="testNG.loginLogoutDataDriveTesintUsingTestNG" tests="6" time="12.942" ignored="1" failures="1" timestamp="06 May 2017 00:10:15 GMT" skipped="0" hostname="LAPTOP-RO5NS03S" errors="0">
  <testcase name="loginLogoutTestWtihIncorrectCredencials" time="0.963" classname="testNG.loginLogoutDataDriveTesintUsingTestNG"/>
  <testcase name="loginLogoutTestWtihIncorrectCredencials" time="0.965" classname="testNG.loginLogoutDataDriveTesintUsingTestNG"/>
  <testcase name="loginLogoutTestWtihIncorrectCredencials" time="0.933" classname="testNG.loginLogoutDataDriveTesintUsingTestNG"/>
  <testcase name="loginLogoutTestWtihIncorrectCredencials" time="0.986" classname="testNG.loginLogoutDataDriveTesintUsingTestNG"/>
  <testcase name="loginLogoutTestWtihIncorrectCredencials" time="8.063" classname="testNG.loginLogoutDataDriveTesintUsingTestNG">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at testNG.loginLogoutDataDriveTesintUsingTestNG.loginLogoutTestWtihIncorrectCredencials(loginLogoutDataDriveTesintUsingTestNG.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- loginLogoutTestWtihIncorrectCredencials -->
  <testcase name="loginLogoutTestWtihIncorrectCredencials" time="1.032" classname="testNG.loginLogoutDataDriveTesintUsingTestNG"/>
  <testcase name="loginLogoutTestWtihCorrectCredencials" classname="testNG.loginLogoutDataDriveTesintUsingTestNG">
    <skipped/>
  </testcase> <!-- loginLogoutTestWtihCorrectCredencials -->
</testsuite> <!-- testNG.loginLogoutDataDriveTesintUsingTestNG -->
